wails-frontend/src/pages/PasswordPromptHandler/Transaction/Transaction.tsx

```diff
@@ -26,15 +33,24 @@ export function Transaction(props: PromptRequestData) {
             <p className="mas-menu-active">
               {Intl.t('password-prompt.sign.from')}
             </p>
-            <p className="mas-menu-default">{Nickname}</p>
+            <p className="mas-menu-default">{maskNickname(Nickname)}</p>
           </div>
           <p className="mas-caption">{maskAddress(WalletAddress)}</p>
         </div>
         <div className="h-8 w-8 rounded-full flex items-center justify-center bg-neutral">
           <FiArrowRight size={24} className="text-primary" />
         </div>
         <div className="flex flex-col">
-          <p className="mas-menu-active">{Intl.t('password-prompt.sign.to')}</p>
+          <div className="flex gap-2">
+            <p className="mas-menu-active">
+              {Intl.t('password-prompt.sign.to')}
+            </p>
+            {RecipientAddress ? (
+              <p className="mas-menu-default">
```

Comment:
I believe the `truncate` class achieve the same result with no dependencies. 🤔 

```suggestion
              <p className="mas-menu-default truncate">
```

Reply:
no, it doesn't work